### ISSUE TITLE
Questions about experimental settings in robustness test

### ISSUE THREAD

**Issue by Wuziyi616** [cuge1995/PointCutMix#1]
Hello! Thanks for your great work. I am quite curious about your experiments in **Sec 4.4 Robustness test**. As you cited my work IF-Defense as a baseline in your paper, I wonder if you used the same setting as mine? For example in the **Table 2** results *PointCutMix-K*, was it obtained by dropping against the DGCNN model trained using *PointCutMix-K*? Or did you first generate adversarial point clouds on the *Baseline* DGCNN, and then tested the *PointCutMix-K* pre-trained DGCNN on them?

Also how did you obtain the **Table 3** results? Did you first apply point dropping attack on the baseline DGCNN weight provided by my repo, and then tested DGCNN trained using your proposed augmentation methods on these generated adversarial point clouds? It would great if you can kindly answer my questions. Thanks!

**Comment by cuge1995** [comment]
Hey, thank you for your attention. 

Firstly, about Table 2, Yes, you're right, we use the DGCNN model trained using PointCutMix-K, and then use the code your open-sourced to attack the model, and finally inference it to get the acc. 

Secondly, about Table 3, we first apply point dropping attack based on the pretrained model you're provided, and attack it, and then tested DGCNN trained using our proposed augmentation methods on these generated adversarial point clouds.

We will update it in the next version of the paper for a more clear description.
Anyway, thanks for open-source the baseline attack test.

**Comment by Wuziyi616** [comment]
Thank you for your quick reply. Yes, I agree that they are correct evaluation settings, and the performance of your method is really impressive. Are you considering adding kNN or other attacks in the future? You mentioned the normalization problem in the paper, but I think a simple workaround is to clip the points that move outside of the unit ball to the surface of the ball. Just a small suggestion :)

**Comment by cuge1995** [comment]
Thanks for your suggestion

**Comment by PaulTHong** [comment]
Hey，author cuge1995. I'm still curious about your description. What's the difference between Table 2 and 3? Could you help me to explain it again?  You mentioned:
'Firstly, about Table 2, Yes, you're right, we use the DGCNN model trained using PointCutMix-K, and then use the code your open-sourced to attack the model, and finally inference it to get the acc.'
How to attack the model? Using which script?

I can't figure out the difference between the implementations of Table 2 and 3.

Thank you very much!

**Comment by PaulTHong** [comment]
Or is the difference just reflected on the selected model of generating adversarial samples: base DGCNN (Table 3)  or PointCutMix-trained DGCNN (Table 2)?

**Comment by cuge1995** [comment]
Thank you for your attention.
Table 2 and 3 of the current version differ from what we discussed before.

**Comment by PaulTHong** [comment]
Yeah, I know this point. You have updated a new version. My key is the difference between your stated robustness and defense (corresponding to 2 tables in the manuscript with different numerical results). Can you help me? Thanks again.

**Comment by cuge1995** [comment]
One is under the drop point attack, this is a white-box attack.
Another is under a transfer-based black-box attack.

**Comment by PaulTHong** [comment]
Oh, I see it.  My understanding is the difference just reflects on the selected model of generating adversarial samples: base DGCNN (provided by the IF-Defense) or PointCutMix-trained DGCNN. The other steps are all the same (test on generated samples with PointCutMix-trained DGCNN). Am I right?

**Comment by cuge1995** [comment]
Yes

**Comment by PaulTHong** [comment]
Got it. Thank you very much!
